src/components/Checkout/LoanPrice.vue

```diff
@@ -4,10 +4,13 @@
 			<select
 				v-model="selectedOption"
 				class="loan-price medium-text-font-size"
-				@change="updateLoanReservation()">
-				<option v-for="price in prices"
-					:key="price"
-					:value="price">${{ price }}
+				@change="updateLoanReservation()"
+			>
+				<option v-for="priceOption in prices"
+					:key="priceOption"
+					:value="priceOption"
+				>
+					${{ priceOption }}
```

Comment:
I've updated price to priceOption in this context as we already have a 'price' prop coming in and were getting a warning for having a 2nd price variable declaration/assignment.